lib/policial/limits_checker.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'logger'
+
+module Policial
+  # Public: Checks number of files received from GitHub and notifies if user
+  # reached GitHub per_page limit.
+  class LimitsChecker
+    DEFAULT_PERPAGE = 30
```

Comment:
Can't we get this value from Octokit itself?

Reply:
Unfortunately no, seems like there is no any references inside octokit gem.
https://github.com/octokit/octokit.rb/blob/8404007e9e11d3695df0907adcb3514decd23287/lib/octokit/configurable.rb#L44